app/controllers/user/UserStats.controller.ts

```diff
@@ -84,14 +103,7 @@ export async function create(request: IUserRequest, response: Response) {
  * @apiName GetCoinsOfUser
  * @apiGroup User
  * 
- * @apiParam {string} user The id of the user. // Not sure about this param. Might be all users since there's no path param?
- * 
- * @apiSuccess {number} coins The number of coins the user has.
- * 
- * @apiSuccessExample {json} Success-Response:
- *      {
- *          "coins": 3,
- *      }
+ * @apiSuccess {string} coins The number of coins the user has.
```

Comment:
I changed this parameter to a string because you said it's a string-based response, but why wouldn't `coins` return as a number since that's what it is in `UserStats.ts`?

Reply:
All raw responses over the web are strings.